packages/plugin-core/src/showcase/SettingsUITip.ts

```diff
@@ -0,0 +1,45 @@
+import { assertUnreachable } from "@dendronhq/common-all";
+import { ShowcaseEntry } from "@dendronhq/engine-server";
+import * as vscode from "vscode";
+import { showMeHowView } from "../views/ShowMeHowView";
+import {
+  DisplayLocation,
+  IFeatureShowcaseMessage,
+} from "./IFeatureShowcaseMessage";
+
+export class SettingsUITip implements IFeatureShowcaseMessage {
+  shouldShow(displayLocation: DisplayLocation): boolean {
+    switch (displayLocation) {
+      case DisplayLocation.InformationMessage:
+      case DisplayLocation.TipOfTheDayView:
+        return true;
+      default:
+        assertUnreachable(displayLocation);
+    }
+  }
+  get showcaseEntry(): ShowcaseEntry {
+    return ShowcaseEntry.SettingsUI;
+  }
+
+  getDisplayMessage(_displayLocation: DisplayLocation): string {
+    return `Modify Dendron Config using Dendron: Configure(UI) command`;
```

Comment:
Let's update this to 

"You can configure Dendron by using the 'Dendron: Configure(UI) command'. You can also optionally edit the config file directly it with the `Dendron: Configure(yaml) command`.